test/scripts/make-kind-cluster.sh

```diff
@@ -77,6 +82,35 @@ do
     kind::cluster::load "$image"
 done
 
+if [[ "${MULTINODE_CLUSTER}" == "true" ]]; then
+    # Install metallb.
+    ${KUBECTL} apply -f https://raw.githubusercontent.com/metallb/metallb/master/manifests/namespace.yaml
+    if ! kubectl get secret -n metallb-system memberlist; then
+        ${KUBECTL} create secret generic -n metallb-system memberlist --from-literal=secretkey="$(openssl rand -base64 128)"
+    fi
+    ${KUBECTL} apply -f https://raw.githubusercontent.com/metallb/metallb/master/manifests/metallb.yaml
```

Comment:
probably makes sense to pin a tagged version of metallb, WDYT?

Reply:
good point!